src/wp-includes/class-wp-block-type-registry.php

```diff
@@ -96,6 +96,15 @@ public function register( $name, $args = array() ) {
 
 		$this->registered_block_types[ $name ] = $block_type;
 
+		/**
+		 * Fires after a block type is registered.
+		 *
+		 * @since 6.1.0
+		 *
+		 * @param WP_Block_Type|false $block_type The registered block type on success, or false on failure.
```

Comment:
From what I can see this won't be called on failure as the function returns early. It seems fine to only call the action on success though.

Reply:
Something else that came to mind is that 'Fires after a block type is registered.' might not be specific enough. It might be worth mentioning that it's specifically when a block is registered with the WP_Block_Type_Registry class.

The reason I'm thinking this is that there's the JavaScript `registerBlockType` function, and so documentation readers might get confused about the timing of this action.

Same would apply for the unregister action too.

Reply:
I've updated both of the hooks to remove `|false` and the reference to failure, and updated the description to flag `with the WP_Block_Type_Registry class.`